File: rock_paper_scissors.py
import random
'''I am going to define a function that the program will use to produce 
pseudo-random result .
''' 
choice_list = ["Scissors", "Rock", "Paper"]
def programs_pick():
    return random.choice(choice_list) 

test = programs_pick()
print(test)
print("\t\t\t\tWe are going to play a game of rock, paper and scissors\n\n")
print("\t\t\t\t\t\t\tHave fun\n")

your_score = int()
progam_score = int()

while True:
    program_choice = programs_pick().upper()
    your_choice = input("Pick among (ROCK, PAPER SCISSORS): ").upper()
    if your_choice == "ROCK" and program_choice == "PAPER":
        print(f'I picked {program_choice}')
        print("You Lost!!!")

    elif your_choice == "ROCK" and program_choice == "SCISSORS":
        print(f'I picked {program_choice}')
        print("You Won!!!")
        your_score += 1

    elif your_choice == "ROCK" and program_choice == "ROCK":
        print(f"I picked {program_choice}")
        print("It's a draw :) ")

    elif your_choice == "SCISSORS" and program_choice == "SCISSORS":
        print(f"I picked {program_choice}")
        print("It's a draw :) ")

    elif your_choice == "SCISSORS" and program_choice == "ROCK":
        print(f"I picked {program_choice}")
        print("You lost!!!")

    elif your_choice == "SCISSORS" and program_choice == "PAPER":
        print(f"I picked {program_choice}")
        print("You won!!!")
        your_score += 1

    elif your_choice == "PAPER" and program_choice == "PAPER":
        print(f"I picked {program_choice}")
        print("It's a draw :) ")

    elif your_choice == "PAPER" and program_choice == "ROCK":
        print(f"I picked {program_choice}")
        print("You won!!!")
        your_score += 1

    elif your_choice == "PAPER" and program_choice == "SCISSORS":
        print(f"I picked {program_choice}")
        print("you lost!!")
    
    to_continue = input("Do you want to continue?(yes/no) ").lower()
    if to_continue == "yes":
        continue
    else:
        break
print(f"your score: {your_score}")
if your_score > 1:
    print("You won the game")
else:
    print("Too BAD, you lost the game")

File: RockPaperScissors.java
import java.util.Scanner;
import java.util.Random;
public class RockPaperScissors{

    public static void main(String args[]){

        Scanner scan = new Scanner(System.in);
        System.out.println("Would you like to play a game of Rock Paper Scissors");
        String response = scan.nextLine().toLowerCase();
        if(response.equals("yes")){
            System.out.println("\nGreat!!!\n");
            System.out.println("When I say shoot, you type in your choice\n");
            System.out.println("rock paper scissors.....shoot!!!\n");

            String yourPick = scan.nextLine().toLowerCase();
            String computerChoice = ComputerChoice();

            if(yourPick.equals(computerChoice)){
                System.out.println("It's a tie");
            }else if((yourPick.equals("rock")&&(computerChoice.equals("paper")))){
                System.out.println("Nice move, you won");
            }else if((yourPick.equals("paper")&&(computerChoice.equals("rock")))){
                System.out.println("I win!!, Well better luck next time");
            }else if((yourPick.equals("rock")&&(computerChoice.equals("scissors")))){
                System.out.println("Nice move, you won");
            }else if((yourPick.equals("scissors")&&(computerChoice.equals("rock")))){
                System.out.println("I win!!, well better luck next time");
            }else if((yourPick.equals("scissors")&&(computerChoice.equals("paper")))){
                System.out.println("Nice move, you won");
            }else if((yourPick.equals("paper")&&(computerChoice.equals("scissors")))){
                System.out.println("I win!!, well better luck next time");
            }else System.out.println("Sorry!!!, will update to include possibility");
            
            System.out.println("Incase your curious I picked "+computerChoice);
        }else
            System.out.println("Darn!!!. Next time I suppose\n");
            System.exit(0);
        scan.close();
    }


    public static String ComputerChoice(){
        int num;
        Random random = new Random();
        num = random.nextInt(3);
        switch(num){
            case 0: return "rock";
            case 1: return "paper";
            case 2: return "scissors";
            default: return " ";
        }
    }
}